plugins/woocommerce/package.json

```diff
@@ -58,6 +58,7 @@
 		"test:e2e:blocks": "pnpm --filter='@woocommerce/block-library' test:e2e",
 		"test:e2e:with-env": "bash ./tests/e2e-pw/run-tests-with-env.sh",
 		"test:perf": "./k6 run ./tests/performance/tests/gh-action-pr-requests.js",
+		"test:plugincheck": "wp-env run tests-cli wp plugin check woocommerce",
```

Comment:
I added this in because otherwise it was trying to run the command with a filter and that was causing it to fail.